fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -120,7 +120,12 @@ ACTOR Future<Void> getRate(UID myID, Reference<AsyncVar<ServerDBInfo>> db, int64
 			reply = Never();
 			*outTransactionRate = rep.transactionRate;
 			*outBatchTransactionRate = rep.batchTransactionRate;
-			//TraceEvent("MasterProxyRate", myID).detail("Rate", rep.transactionRate).detail("BatchRate", rep.batchTransactionRate).detail("Lease", rep.leaseDuration).detail("ReleasedTransactions", *inTransactionCount - lastTC);
+			TraceEvent("MasterProxyTxRate", myID)
```

Comment:
I think has the potential to be a bit spammy, and it also may be better if it is in a more discoverable place, like `ProxyMetrics`.